### PR TITLE
doc: Add pradeepcrao to sec release contributors

### PR DESCRIPTION
**Pull request by yanavlasov**
Signed-off-by: Yan Avlasov <yavlasov@google.com>

### CODE REVIEW

**Comment by alyssawilk** [comment]
we've traditionally not updated the file for transient additions such as release managers and patch fixers - do we think it's worth the churn?

File: OWNERS.md [hunk 1]
```diff
@@ -79,6 +79,9 @@ contributors to envoy-setec and relevant Slack channels from:
   * Alessandro Gario ([alessandrogario](https://github.com/alessandrogario))
   * Mike Myers ([mike-myers-tob](https://github.com/mike-myers-tob))
 
+* [Feb 2022 Security Release Contributors]
+  * Pradeep Rao ([pradeepcrao](https://github.com/pradeepcrao)) expiring 2/28/2022
```
Comment (htuch) [inline OWNERS.md:83]: 2022 or 2021? Either is good, since I believe @pradeepcrao will be contributing valuable security fixes over the next year.

Comment (RyanTheOptimist) [reply OWNERS.md:83]: 2021 would be in the past, right?

Comment (htuch) [reply OWNERS.md:83]: Sorry, brain fart, I think we should make it 2023 then.

Comment (yanavlasov) [reply OWNERS.md]: Done.

Comment (alyssawilk) [reply OWNERS.md]: I think if we're making it 2023 we just add him to the setec group, as discussed on #envoy-security-team.  Ditto @RyanTheOptimist as long as we have the file open?

Comment (alyssawilk) [reply OWNERS.md]: /wait

Comment (yanavlasov) [reply OWNERS.md]: Done

File: OWNERS.md [hunk 2]
```diff
@@ -79,6 +80,10 @@ contributors to envoy-setec and relevant Slack channels from:
   * Alessandro Gario ([alessandrogario](https://github.com/alessandrogario))
   * Mike Myers ([mike-myers-tob](https://github.com/mike-myers-tob))
 
+* [Feb 2022 Security Release Contributors]
```
Comment (alyssawilk) [inline OWNERS.md:83]: I think we're just adding them both to EST no?

Comment (alyssawilk) [reply OWNERS.md:83]: /wait

Comment (alyssawilk) [reply OWNERS.md:85]: Cool.  LGTM but can you get a thumbs up from a non-googler on setec for this one?

**Pull request closed without merging**